webpack.config.js

```diff
@@ -48,4 +57,7 @@ module.exports = {
     modules: ['src', 'node_modules'],
   },
   devtool: 'sourcemap',
+  node: {
+    __dirname: false,
+  },
```

Comment:
[isn't `__dirname: false` the default](https://webpack.js.org/configuration/node/#node__dirname)?

It may be good to put an inline comment here to briefly mention why it is important, given that we are explicitly setting it.

Reply:
I think it is but last time I checked, it was not working as intended. Let me check again.

Reply:
ha well, it works now.. 🤷